application/pr/viewer.jsx

```diff
@@ -77,6 +77,22 @@ export default class PRViewer extends React.Component {
 
     }
 
+    // Non-participant licensing commitments
+    askNPLC (ev) {
+        let st = this.state;
+        fetch(pp + "api/get-repo/" + [st.owner, st.shortName].join("/"), { credentials: "include"})
+            .then(utils.jsonHandler)
+            .then((data) => {
+                let repoId = data.id;
+                let nplcUrl = new URL(['/2004/01/pp-impl/nplc', repoId, st.num, 'edit'].join("/"), 'https://www.w3.org/');
+                let qs = st.pr.contributors.map((c) => {return 'contributors[]=' + c;}).concat(st.pr.groups.map((g) => {return 'groups[]=' + g;})).join('&');
+                nplcUrl.search = qs;
+                window.location = nplcUrl.toString();
```

Comment:
rather than a redirect, given that this moves the user out of the existing app, I think a link would be best here (maybe defaulting to open in a separate tab)

Reply:
Done in https://github.com/w3c/ash-nazg/pull/163/commits/c5291ccd6ce483b2e738e7c2d15962dae6776e19

I added the link in the list of fixes recommended by the repo-manager instead of a button at the top of the PR

Reply:
sorry to be annoying, but I think having it looking like a button and listed with the other options would be much easier for staff contacts to find

Reply:
Not at all, I misinterpreted your comment. It should be fixed in https://github.com/w3c/ash-nazg/pull/163/commits/03ed3c338ba33c0d9c333dbfee0d5c22fd519f6e